### PR TITLE
Update standards.md

### PR DESCRIPTION
**Pull request by chrisdburr**
Added changes from @S-Laher to standards guidance.

### CODE REVIEW

File: site/docs/introductory-resources/standards.md [hunk 1]
```diff
@@ -66,15 +68,21 @@ These standards often form the baseline language utilised in other standards sup
...
+    - [AI Standards Hub Link](https://aistandardshub.org/ai-standards/information-technology-artificial-intelligence-management-system/)
+
+- **DCB0129:** Clinical Risk Management: its Application in the Manufacture of Health IT Systems & **DCB0160:** Clinical Risk Management: its Application in the Deployment and Use of Health IT Systems
+    - These standard form a pair of clinical risk management standards implemented by the NHS which are enforced under section 250 of the Health and Social Care Act 2012 to assure the clincal safety of Health Information Technology (HIT) systems
```
Comment (cptanalatriste) [inline site/docs/introductory-resources/standards.md:82]: These standardS?

File: site/docs/introductory-resources/standards.md [hunk 2]
```diff
@@ -66,15 +68,21 @@ These standards often form the baseline language utilised in other standards sup
...
+    - [DCB0129](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/dcb0129-clinical-risk-management-its-application-in-the-manufacture-of-health-it-systems) is for organisations who develop and maintain HIT that will be utilised in health and care environments.
```
Comment (cptanalatriste) [inline site/docs/introductory-resources/standards.md:83]: Not sure if HIT was defined before...

File: site/docs/introductory-resources/standards.md [hunk 3]
```diff
@@ -83,42 +91,63 @@ These standards ensure specified thresholds have been achieved and enable entiti
...
 **Examples**
 
-- W3C - SL to do
-- Example 2
-::: -->
+- IEEE 2941: Standard for Artificial Intelligence (AI) Model Representation, Compression, Distribution, and Management
+    - The AI development interface, AI model interoperable representation, coding format, and model encapsulated format for efficient AI model inference, storage, distribution, and management are discussed in this standard.
+    - [AI Standards Hub Link](https://aistandardshub.org/ai-standards/ieee-standard-for-artificial-intelligence-ai-model-representation-compression-distribution-and-management/)
+
+- HTML (HyperText Markup Language)
```
Comment (cptanalatriste) [inline site/docs/introductory-resources/standards.md:146]: Not sure I would call HTML a standard...

File: site/docs/introductory-resources/standards.md (hunk 1 above)
Comment (chrisdburr) [reply site/docs/introductory-resources/standards.md:82]: Yep. Thanks!

**Pull request merged**